packages/components/src/components/hds/super-select/index.hbs

```diff
@@ -0,0 +1,30 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{! NOTE: naming is temporary to not conflict with existing power-select wrapper }}
+<div class={{this.classNames}}>
+  {{!
+    Required:
+    * onChange
+    * options or search
+  }}
+  <PowerSelect
+    @ariaDescribedBy={{@ariaDescribedBy}}
+    @ariaInvalid={{@ariaInvalid}}
+    @ariaLabel={{@ariaLabel}}
+    @ariaLabelledBy={{@ariaLabelledBy}}
+    @disabled={{@disabled}}
+    @onChange={{@onChange}}
+    @options={{@options}}
+    @renderInPlace={{@renderInPlace}}
```

Comment:
we know that some of these arguments should have a different default to make the compliant, for example, `renderInPlace` should be `true`. It's probably worth exploring changing such defaults in this spike.